.changeset/curvy-pumpkins-carry.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-admin": patch
+---
+
+Prevent `ExternalLinkBlock` when the url field is empty
```

Comment:
This doesn’t sound right… „Prevent … validation“?

Reply:
I meant

```suggestion
Prevent error in `ExternalLinkBlock` when the url field is empty
```